x-pack/plugins/apm/public/components/shared/DatePicker/index.tsx

```diff
@@ -7,10 +7,9 @@
 import { EuiSuperDatePicker } from '@elastic/eui';
 import { isEmpty, isEqual, pickBy } from 'lodash';
 import React from 'react';
-import { useHistory } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
```

Comment:
Replacing our `useLocation` here with the one from `react-router-dom`. This is also fixed in #76343 but CSM is not using our old location provider.